### ISSUE TITLE
1 badge icon if I'm not signed-in Microsoft account to sync

### ISSUE THREAD

**Issue by frederikhors** [microsoft/vscode#94822]
Please remove the `1` badge if I not Sign-In Microsoft account to sync.

Please.

![image](https://user-images.githubusercontent.com/41120635/78927648-89e38700-7a9f-11ea-93a7-d3f5813073e5.png)

Version: 1.45.0-insider (user setup)
Commit: 2225d77536d12fea2eda04930f431a5eb0930bba
Date: 2020-04-09T09:22:11.706Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Comment by bergmeister** [comment]
Yes please, this is very confusing as it looks like an update is available

**Comment by sandy081** [comment]
I see that the number badge over this has been used for updates and since settings sync feature is also available under this gear icon, notifications from settings sync feature are also shown here. So number badges are shown when

- there is an update available
- there are sync conflicts
- sync requires user sign in

I think what is missing here when `sync requires user sign in` badge is shown, is to have a turn off sync action. So that user can either sign in to continue sync or stop syncing.

**Comment by sandy081** [comment]
/duplicate

https://github.com/microsoft/vscode/issues/94888

There was a bug in insiders before that enabled Settings Sync automatically and we fixed this in next insiders here - https://github.com/microsoft/vscode/issues/94888. But fix cannot recover from disabling the sync and removing the badge. 

To fix this please disabled Sync using the command  `Preferences Sync: Turn off`

**Comment by bergmeister** [comment]
Thanks for the detailed info @sandy081 , worked for me :-)
